api-playground/openapi-setup.mdx

```diff
@@ -4,7 +4,7 @@ description: "Reference OpenAPI endpoints in your docs pages"
 icon: "file-json"
 ---
 
-OpenAPI is a specification for describing REST APIs. Mintlify supports OpenAPI 3.0+ documents to generate interactive API documentation and keep it up to date.
+OpenAPI is a specification for describing APIs. Mintlify supports OpenAPI 3.0+ documents to generate interactive REST API documentation, and OpenAPI 3.1+ for documenting webhooks.
```

Comment:
I think the disclaimer on the version should be closer with where the content is at. I don't think it needs to be at the high level overview